### ISSUE TITLE
Calendar is not being displayed in Edge

### ISSUE THREAD

**Issue by KongBorup** [KongBorup/rollespilsfabrikken-forum-calendar-frontend#18]
**Version:** v26542626

When the calendar page ([`https://forum.rollespilsfabrikken.dk/#/kalender`](https://forum.rollespilsfabrikken.dk/#/kalender)) is opened in Edge, the calendar area is blank and the following error is shown in the console:
```
RangeError: Array length must be a finite positive integer
```
If you attempt the switch month, a far more verbose error is thrown:
```js
[object Error]
{
  description: "Array length must be a finite positive integer"
  message: "Array length must be a finite positive integer"
  number: -2146823259
  stack: "RangeError: Array length must be a finite positive integer at Rt (https://forum.rollespilsfabrikken.dk/js/chunk-vendors.c22be719.js:7:16954) at Anonymous function (https://forum.rollespilsfabrikken.dk/js/app.57774800.js:1:85470) at e.prototype._render (https://forum.rollespilsfabrikken.dk/js/chunk-vendors.c22be719.js:7:23724) at r (https://forum.rollespilsfabrikken.dk/js/chunk-vendors.c22be719.js:7:27812) at nr.prototype.get (https://forum.rollespilsfabrikken.dk/js/chunk-vendors.c22be719.js:7:30670) at nr.prototype.run (https://forum.rollespilsfabrikken.dk/js/chunk-vendors.c22be719.js:7:31406) at Xn (https://forum.rollespilsfabrikken.dk/js/chunk-vendors.c22be719.js:7:29622) at Anonymous function (https://forum.rollespilsfabrikken.dk/js/chunk-vendors.c22be719.js:7:12759) at ct (https://forum.rollespilsfabrikken.dk/js/chunk-vendors.c22be719.js:7:12152) at a (https://forum.rollespilsfabrikken.dk/js/chunk-vendors.c22be719.js:13:147979)"
}
```
Full window screenshot:
![image](https://user-images.githubusercontent.com/16561050/85198173-beb55b00-b2e6-11ea-9611-1d0304cc3b90.png)

**Comment by KongBorup** [comment]
Logging the return value of the computed `gridInfo` property for the `CalendarGrid` component shows:
```js
[object Object]: {firstDay: NaN, numDays: 31, numLeftOverFields: NaN, numWeeks: NaN}
```
This is likely the reason that the error is thrown during Vue's render as the `NaN`-values are used for looping as such:
```vue
<li v-for="date in gridInfo.numDays">
```
Figuring out why those values are `NaN` would probably lead to the solution to this issue.

**Comment by KongBorup** [comment]
The problem is rooted here:
```js
export function getWeekday(...args) {
  const date = monthJump(new Date(args), 1);
  // ...
}
```
In Edge, the `Date` constructor does not work when the input is an Array. Spreading the `args` array should fix the issue: `new Date(...args)`.

**Comment by KongBorup** [comment]
Fixed in a136f00.
